sdk/js-connect/core/base/src/utils/layout/index.ts

```diff
@@ -0,0 +1,36 @@
+export {
```

Comment:
this `utils/layout` is quite a piece of work. I don't see unit tests in this directory. is this unit tested somehow elsewhere in this large diff?

Reply:
There are some tests that use this downstream https://github.com/wormhole-foundation/connect-sdk/blob/develop/core/definitions/__tests__/transferVaa.ts

Reply:
Some tests here https://github.com/wormhole-foundation/wormhole/blob/3926dc88eba6d3e9ede3eed4864796a99a9249fc/sdk/js-connect/core/base/__tests__/layout.ts